packages/cli/src/utils/try.ts

```diff
@@ -0,0 +1,15 @@
+export function tryFunc<T>(func: () => T, ifException: T = null): T {
```

Comment:
I understand the idea behind this code is to use it where there is no need to capture error output but this is pretty much every programming book anti-pattern example 💥

Reply:
Hey, this is basically Ruby's `rescue nil`!

Reply:
First google search says it is [bad](https://rubyinrails.com/2015/08/27/rescue-nil-in-ruby-on-rails/) but I don't know Ruby 🤷‍♂  Anyway I don't mind having it. My point is that I think it is anti-pattern because eventually people misuse it and it cause high cost errors. Let's add it and see what happens 😈